rss2017/process/processToFile.js

```diff
@@ -0,0 +1,41 @@
+var fs = require('fs');
+
+var mainProcessFile = process.argv[2];
+var outputPath = process.argv[3];
+var track = process.argv[4];
+
+console.log('Main process', mainProcessFile);
+
+var mainProcess = fs.readFileSync(mainProcessFile, 'utf8');
+
+// Look for << filename.js>> and replace that string with the code in the file
+var regex = new RegExp('<<(.+\.js)>>', 'g');
+var matches = mainProcess.match(regex);
+console.log('matches', matches);
+if(matches) {
+
+	matches.forEach(function(m) {
+
+		var fileName = m.replace('<<', '').replace('>>', '');
+		var fileContent = fs.readFileSync(outputPath + '/' + fileName, 'utf8');
+		mainProcess = mainProcess.replace(m, fileContent);
+		console.log('Processing file', fileName);
+	})
+}
+
+
+// If there is track information add it as an argument to the function call
+if (track) {
```

Comment:
In RSS there are two tracks (like ICLR).  In order to avoid creating nearly identical copies of the process templates and js files I added the track parameter.  
I was nervous about adding it to the existing processToFile because I didn't want to mess up existing conferences. I like your solution.